handlers.ts

```diff
@@ -0,0 +1,61 @@
+import fs = require("fs");
+
+const filePath = "file.txt";
```

Comment:
create file named `config.json`, read filePath and port from this file

Reply:
done